lib/src/main/kotlin/com/lemonappdev/konsist/core/filesystem/PathProvider.kt

```diff
@@ -36,6 +36,6 @@ class PathProvider(
     companion object {
         private val pathVerifier = PathVerifier()
 
```

Comment:
What is better option:
1) make `PathProvider` class public and use it in  snippet test
2) 'PathProvider' should be still internal and I should copy 
```private val pathVerifier = PathVerifier()
fun getInstance() = PathProvider(KoFileFactory(), ProjectRootDirProviderFactory(pathVerifier))``` to snippet test?